src/containters/Home/components/HeroView/HeroView.js

```diff
@@ -1,39 +1,59 @@
-import React from 'react'
-import './index.scss'
-import { IoIosArrowForward } from 'react-icons/io'
-
+import React from 'react';
+import './index.scss';
+import { IoIosArrowForward } from 'react-icons/io';
+import { useEffect, useState } from 'react';
+import { heroView } from '../../services/heroView';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
 const HeroView = () => {
+    const [heroViewData, setHeroViewData] = useState({});
+
+    useEffect(() => {
+        Promise.all([heroView()])
+            .then(res => {
+                // console.log('heroView', res[0].data);
+                setHeroViewData(res[0].data);
+                console.log(heroViewData);
+            }).catch((error) => {
+                console.log('heroView error', error);
+            });
+    }, []);
     return (
-        <div>
-            <div className="HeroView">
-                <div className="container content-container">
-                    <div className="content">
-                        <p className="hint">
-                            25% off promotional sale
-                        </p>
-                        <h1 className="title">
-                            All you need in the kitchen.
-                        </h1>
-                        <p className="desc">
-                            commodo dolor sit amet cum diceam est un partuient nascetur mas ligula massa consequat est un dolor
-                        </p>
-                        <button className="mainBtn">Browse Products</button>
-                        <div className="assistant">
-                            <div className="content-assistant">
-                                <img src="./assets/assistant.png" alt="assistantImg" />
-                                <span>Talk to an assistant</span>
-                                <span className="icon"><IoIosArrowForward /></span>
+        <Router>
+            <div>
+                <div className="HeroView">
+                    <div className="container content-container">
+                        <div className="content">
+                            <p className="hint">
+                                {heroViewData.promotion_text}
+                            </p>
+                            <h1 className="title">
+                                {heroViewData.title}
+                            </h1>
+                            <p className="desc">
+                                {heroViewData.description}
+                            </p>
+                            <button className="mainBtn">
```

Comment:
- Please remove the button element and use the link directly and give it the same class or styling as the button is not needed and will introduce unexpected behavior